tf/environments/dev/main.tf

```diff
@@ -235,7 +235,7 @@ resource "aws_codestarconnections_connection" "ooniapi" {
 }
 
 resource "aws_codestarconnections_connection" "oonith" {
-  name          = "oonith"
+  name = "oonith"
```

Comment:
You may want to setup your code editor to run `terraform fmt` on save to ensure these kinds of spacings are preserved.

With VSCode I use this extension and it works quite well: https://marketplace.visualstudio.com/items?itemName=HashiCorp.terraform. It also provides type hinting and some basic syntax checking which is quite handy.